### PR TITLE
Jakarta EE 9.1 alignment

### LINKED ISSUE #219: Jakarta EE 9.1 alignment
### For each spec, the following updates need to happen, refer [this doc](https://docs.google.com/document/d/1bhhfmhxHJpQdmdUtQw5o_WtScpzkfHoTNGPvrQnGhc0/edit) for instructions:

- [ ] Create a branch called `jakarta91-branch` off master to perform Jakarta EE 9.1 alignment
- [ ] Only jakarta namespace changes are allowed in the release without any other functional changes with the exception of typos or tck fixes
- [ ] pom.xml: utilise microprofile-parent 2.1 as it has pulled in all of the Jakarta EE 9.1 as well as the necessary TCK dependencies (see [this](https://github.com/eclipse/microprofile-config/pull/663) PR on how to)
- [ ] spec docs: rename Java EE to Jakarta EE, rename javax to jakarta: `find . -type f -name '*.asciidoc' -exec sed -i '' s/javax./jakarta./ {} +`
- [ ] Update all javax to jakarta for all java files `import javax.*` needs to be updated to `jakarta.*`: `find . -type f -name '*.java' -exec sed -i '' s/javax./jakarta./ {} +`
- - [ ] TCKs
- - [ ] APIs
- [ ] Update bnd file:  `find . -type f -name '*.bnd' -exec sed -i '' s/javax./jakarta./ {} +`

- [ ] service loader file javax.* to jakarta.*: `find . -type f -name "javax*" -exec sh -c 'mv "$0" "${0/javax/jakarta}"' '{}' \;
`
- [ ] deliver the changes to both `jakarta91-branch` and `master`
- [ ] Perform a RC release off the branch `jakarta91-branch`

### PR DESCRIPTION
**Pull request by fmhwong**
Fixes #219 

Signed-off-by: Felix Wong <fmhwong@ca.ibm.com>

### CODE REVIEW

File: tck/base/pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <parent>
         <artifactId>microprofile-opentracing-tck-parent</artifactId>
         <groupId>org.eclipse.microprofile.opentracing</groupId>
-        <version>2.1-SNAPSHOT</version>
+        <version>3.0-SNAPSHOT</version>
```
Comment (Emily-Jiang) [inline tck/base/pom.xml:21]: The <build> element from the tck pom.xml should be removed as well, I think.

Comment (fmhwong) [reply tck/base/pom.xml:21]: Just push an update to the commit for removing the `<build>` element in pom.xml

Comment (Emily-Jiang) [reply tck/base/pom.xml:21]: What I meant is to remove `<build>` elment from the tck\pom.xml.

**Review by Emily-Jiang — APPROVED** [review]
LGTM! Thank you @fmhwong

**Pull request merged**